src/TransCreator/Style.js

```diff
@@ -8,20 +10,30 @@ export default StyleSheet.create({
   footerAndroid: {
     backgroundColor: 'transparent',
   },
-  dateItem: {
-    borderBottomWidth: 0,
-    marginTop: 30,
-    marginBottom: 20,
+  dateSection: {
+    flex: 1,
+    flexDirection: 'row',
+    backgroundColor: 'white',
+    width: DeviceWidth,
+    borderTopWidth: 1,
+    borderColor: 'rgb(240,240,240)',
```

Comment:
Here is using rgb while the color below is using hex, it may be better if both use hex

Reply:
fixed